readinglist/views/errors.py

```diff
@@ -101,3 +102,8 @@ def error(context, request):
         content_type='application/json')
 
     return reapply_cors(request, response)
+
+
+@view_config(context=HTTPBadRequest)
+def json_error(context, request):
+    return context
```

Comment:
@Natim @tarekziade @ametaireau I had to add this just for the [test app that I setup here](https://github.com/mozilla-services/readinglist/pull/66/files#diff-c445a1cd6e41331ed421066d9a202915R43)

*(it is not necessary when executing the views via httppie or webtest)*

     Note that using ``cornice.tests.support.CatchErrors`` had no effect.
Any hint ?